src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -1639,6 +2181,243 @@ private static unsafe nuint NarrowUtf16ToAscii_Intrinsified(char* pUtf16Buffer,
             goto Finish;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static unsafe nuint NarrowUtf16ToAscii_Intrinsified_256(char* pUtf16Buffer, byte* pAsciiBuffer, nuint elementCount)
+        {
+            // This method contains logic optimized using vector instructions for x64 only.
+            // Much of the logic in this method will be elided by JIT once we determine which specific ISAs we support.
+
+            // JIT turns the below into constants
+
+            uint SizeOfVector256 = (uint)Vector256<byte>.Count;
+            nuint MaskOfAllBitsInVector256 = (nuint)(SizeOfVector256 - 1);
```

Comment:
These can both be `const`, using `Vector256.Size` directly and declaring `const nuint MaskOfAllBitsInVector256` for the latter.

Reply:
Same for the equivalent case in `Intrinsified_512`